api/HTMLVideoElement.json

```diff
@@ -635,8 +635,10 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false,
-              "impl_url": "https://bugzil.la/1800882"
+              "version_added": "preview",
+              "impl_url": "https://bugzil.la/1800882",
+              "partial_implementation": true,
+              "notes": "The <code>metadata</code> argument of the callback function does not support the following properties: <code>processingDuration</code> (<a href='https://bugzil.la/bug 1908246'>1908246</a>), <code>captureTime</code>, <code>receiveTime</code>, and <code>rtpTimestamp</code> (<a href='https://bugzil.la/bug 1908245'>1908245</a>)."
```

Comment:
This is what we [see in the IDL](https://hg.mozilla.org/integration/autoland/diff/40274941ace1f1271e659138923b1ea6f617c603/dom/webidl/HTMLVideoElement.webidl#l1.12). Presumably this will all be fixed on release, at which point we can remove the note and partial implementation. I think adding this info makes tracking what remains a little easier.